runtime/src/lib.rs

```diff
@@ -125,11 +156,13 @@ pub mod opaque {
     /// Opaque block identifier type.
     pub type BlockId = generic::BlockId<Block>;
 
-    impl_opaque_keys! {
+}
+
+
+impl_opaque_keys! {
         pub struct SessionKeys {
 		pub aura: Aura,
 	    }
-    }
 }
 
 pub use types::*;
```

Comment:
why aren't you using a 4320 ms block time like in the 'master' branch?

Reply:
I have made it similar to master

Reply:
i sent you a message in Discord #datahighway-roaming-mining 28 Dec 2021 to highlight that we'll want to avoid encountering this issue on our parachain as it relates to parachain blocktime vs standalone blocktime https://kusama.polkassembly.io/proposal/53.

it seems we need to use a parachain standard block time for the parachain, rather than what we used on our standalone substrate blockchain, and then we'll have to re-calculation how that affects things in the "DHX Mining & Halving" spreadsheet that was shared with you, as the "Block Production / Mining Time" input in the "halving-v1-p1" tab would change, which affects values in other tabs like the % of DHX total supply issued each year in the "halving-v2-p2" tab, and it affects outputs in tab "halving-v2-p3-block-mining-rate" since if we instead used a block time of 12s instead of 4.32s, then the "Block Reward Issuance Per Day (DHX)" would change from 5000 DHX to 1800 DHX, and in tab "halving-v2-p4-block-reward-dist" the you'll see that since the block reward issuance per year decreases, then there is less available for collator and treasury fees, so the ROI for collators is also reduced by ~64%

Reply:
so our DataHighway parachain should have a `MILLISECS_PER_BLOCK` of 12000 ms